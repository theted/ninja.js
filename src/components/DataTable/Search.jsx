import React from 'react';

const Search = ({ onSearch }) => (
  <div className="p-b-1">
    <input
      type="search"
      className="form-control"
      placeholder="SÃ¸g brugere"
      onChange={onSearch}
    />
  </div>
);

export default Search;
