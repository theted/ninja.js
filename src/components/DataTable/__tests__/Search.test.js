import React from 'react';
import { shallow } from 'enzyme';
import Search from '../Search';

const props = {
  onSearch: () => {},
};

describe('Search component', () => {
  it('renders without crashing', () => {
    shallow(<Search {...props} />);
  });

  it('matches snapshot', () => {
    const component = shallow(<Search {...props} />);
    expect(component).toMatchSnapshot();
  });
});
