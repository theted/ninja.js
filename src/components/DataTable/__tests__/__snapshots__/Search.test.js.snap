// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component matches snapshot 1`] = `ShallowWrapper {}`;
